### ISSUE TITLE
Venues

### ISSUE THREAD

**Issue by tomtom5152** [nottinghamtec/Main-Website#27]
One thing that I think would be cool to do is build a public catalog of venues we have worked in, both for a "this is what we can do" thing, and for bigger spaces such as Studio 7, a "this is what can be done in this specific space" type thing.

The idea came about after speaking with Alex Daniel where he wasn't happy for Conferences to use his photos for publicising S7, although whoever does it would need to check with him, TEC branded pages would be a much better thing for both TEC and Alex (as we can control credits easier). Sending people a link to an unlisted page that just showcases the space a little would be a very handy thing to have.

**Comment by samozzy** [comment]
> **Frequent venues**
> There are some spaces that we work in very frequently, being based at the University. These range from small areas like the Students' Union bar and halls of residence, to larger rooms like the Great Hall, all the way up to a retired Carlton TV studio on a nearby satellite campus. We have the tools to accommodate events in any space, and don't limit ourselves to the campuses either.

Then photos and blurby bits on these venues. I would include: Mooch, Den and Live, Studio 7, any halls that have nice photos (potentially one big and one small), Great Hall, the Downs, and maybe some external venues like the Atrium, Colwick and Donington.

**Comment by harry-bridge** [comment]
Have a look at [this](https://github.com/nottinghamtec/Main-Website/tree/Venues), it's a start to see what people think (as you will notice it's all the same venue and picture, that's because I'm really _really_ lazy)

Find it under /about/venues

**Comment by tomtom5152** [comment]
I'm not overly keen on using datafiles for this. I would recommend creating each venue as it's own page as it creates much greater flexibility. I only used the data directory for committee as it I deliberately wanted the flat uniform look that it gives you.

**Comment by harry-bridge** [comment]
It's quite nice to see each venue on one page and then also a detail view on a separate page I thought. Is it possible to do that with a load of single pages without a data file?

**Comment by tomtom5152** [comment]
Yeah fully, you want a custom archetype and then a new layout for the archetype/summary.html. The committee used to be done like this and I changed to the data file option in 638eaa3 due to the simplicity of what I wanted to show (and to prevent having a dedicated page for each person).

**Comment by harry-bridge** [comment]
Ok awesome, I'll give it a go

**Comment by harry-bridge** [comment]
[Venues](https://github.com/nottinghamtec/Main-Website/tree/Venues) has been updated to use archetypes, now someone needs to write some lovely words about each venue.

**Comment by samozzy** [comment]
This looks great. Thanks, Harry.
I have a navigation concern, as there's no (easy) way to get back to the venues page when on a venue, nor is there navigation to the rest of the website. I know that clicking the photo at the top goes back, but I think more is needed. I'll have a think and maybe make something.

I'm also wondering if you had a plan for the dotty button on each venue's menu card?

**Comment by harry-bridge** [comment]
Hit the handy dandy back button in your browser? I could add some breadcrumbs if you wanted, and that was left over from when it was a pop up card, should probably fix that.

Also this was meant to be a client only page as a kind of portfolio that's why it has no links to the rest of the site

**Comment by davidtaylorhq** [comment]
@pickleshb Back button works fine, until someone sends you a link to a specific venue in an email or similar

**Comment by samozzy** [comment]
I agree with David.

I also think the venue name should be on the venue page, and higher and larger than the event name.

**Comment by harry-bridge** [comment]
So do I, and feel free to play around with it if you want

**Comment by samozzy** [comment]
Slideshow: Would like progress bars and next/prev buttons.

**Comment by harry-bridge** [comment]
Are the buttons not enough for you?

<img width="1162" alt="screen shot 2016-03-30 at 15 17 04" src="https://cloud.githubusercontent.com/assets/5897290/14145118/8483d672-f68a-11e5-803e-dfe7a8ff77fe.png">

**Comment by samozzy** [comment]
Nope. :stuck_out_tongue:

**Comment by davidtaylorhq** [comment]
Less is more

**Comment by harry-bridge** [comment]
@davidtaylorhq I don't think materialize has the option for arrows in it's sliders, the dots are enough for me

**Comment by davidtaylorhq** [comment]
I agree, I think the dots are enough (not that I've actually pulled & built it because I'm lazy)

**Comment by samozzy** [comment]
Interestingly, your screenshot has dots in the picture and blue. Mine are below the picture and green.

**Comment by davidtaylorhq** [comment]
@samozzy I blame firefox (at a guess)

**Comment by samozzy** [comment]
I am 97% certain it is unrelated to the new header.
![dotty](https://cloud.githubusercontent.com/assets/15388319/14161315/7adf68e8-f6da-11e5-9ed3-37a5010a22bc.png)

**Comment by harry-bridge** [comment]
It is you need to compile your sass man

**Comment by samozzy** [comment]
@pickleshb But I didn't even edit any CSS...

**Comment by harry-bridge** [comment]
@samozzy Yes but I did, and the main css file isn't in version control, so you'll need to compile it and everything will look pretty

**Comment by samozzy** [comment]
@pickleshb Ahhh this makes sense.

**Comment by johnathan99j** [comment]
Made it so that all venues have there own photos so there's no more placeholders any more. Words are still required in some places. Also added the Albert Hall. This also ties to #55
